src/main/resources/migrations.xml

```diff
@@ -5759,4 +5759,34 @@
 		<!-- Rolling back is not useful anymore -->
 	</changeSet>
 
+	<changeSet id="add-entityAvatars-table" author="fdelcastillo">
+		<createTable tableName="entityAvatars">
+			<column name="entityUuid" type="${uuid_type}">
```

Comment:
In places where we use a "generic foreign key", we have _two_ columns, one for the relatedObjectType and one for the relatedObjectUuid. I suggest to follow this convention.
[Note that even though the relatedObjectType is not always used, it _does_ make it easier to find to which entity this avatar belongs (no need to check each and every possible table.]